### PR TITLE
Phone country fix

### PR DESCRIPTION
**Pull request by guillim**

### CODE REVIEW

File: packages/twenty-server/src/database/commands/upgrade-version/0-40/0-40-upgrade-version.command.ts [hunk 1]
```diff
@@ -38,19 +38,19 @@ export class UpgradeTo0_40Command extends ActiveWorkspacesCommandRunner {
       'Running command to upgrade to 0.40: must start with phone calling code otherwise SyncMetadata will fail',
     );
 
-    await this.recordPositionBackfillCommand.executeActiveWorkspacesCommand(
+    await this.phoneCallingCodeCreateColumnCommand.executeActiveWorkspacesCommand(
```
Comment (Weiko) [inline packages/twenty-server/src/database/commands/upgrade-version/0-40/0-40-upgrade-version.command.ts:41]: why changing the order?

File: packages/twenty-server/src/database/commands/logger.ts [hunk 2]
```diff
@@ -5,42 +5,48 @@ interface CommandLoggerOptions {
   constructorName: string;
 }
 
+/**
+ * The CommandLogger class wraps the NEST Logger instance
```
Comment (Weiko) [inline packages/twenty-server/src/database/commands/logger.ts:9]: We don't use JSdoc in the codebase

Comment (guillim) [reply packages/twenty-server/src/database/commands/logger.ts:9]: oki

File: packages/twenty-server/src/database/commands/upgrade-version/0-40/0-40-upgrade-version.command.ts (hunk 1 above)
Comment (guillim) [reply packages/twenty-server/src/database/commands/upgrade-version/0-40/0-40-upgrade-version.command.ts:41]: not intended.  the merge suggestion happened this way

**Review by Weiko — APPROVED** [review]

**Pull request merged**